docker/lr-align/prepare_run.py

```diff
@@ -16,7 +16,7 @@
     hc = hashlib.sha1(",".join(fqs).encode()).hexdigest()[0:8]
```

Comment:
Is it possible to add some documentation for this python script, since you are touching it here.
It took me sometime to understand what is being accomplished here.